storage/mysql/tree_storage.go

```diff
@@ -104,6 +104,20 @@ func expandPlaceholderSQL(sql string, num int, first, rest string) string {
 	return strings.Replace(sql, placeholderSQL, parameters, 1)
 }
 
+// clearAllStmt clean up all sql.Stmt in cache
+func (m *mySQLTreeStorage) cleanAllStmt() {
+	m.statementMutex.Lock()
+	defer m.statementMutex.Unlock()
+
+	for _, ns := range m.statements {
+		for _, s := range ns {
+			s.Close()
```

Comment:
Please check the error returned from Close. Maybe not much more to do other than log the failure, but I don't like letting errors like this simply disappear.

Reply:
fixed